Program.cs

```diff
@@ -12,16 +80,30 @@ static void Main(string[] args)
                 // Your solution goes here
                 // You can get all needed inputs from task.[Property]
                 // Good luck!
-                string UserLongitude = task.UserLongitude;
-                string UserLatitude = task.UserLatitude;
                 int placesAmount = task.DefibliratorStorages.Length;
 
+                Coordinates userCoordinates = new Coordinates();
+                userCoordinates.LoadFromString(task.UserLongitude, task.UserLatitude);
+
+                Storage nearestStorage = new Storage();
```

Comment:
No need to create a dummy object, you can initialize ```nearestStorage``` to ```null```. You'll have to add a null-check if you want to be extra careful, though. Kinda a matter of preference, at least if we're not talking about performance critical code.